modules/chain/ChainConfig.ts

```diff
@@ -0,0 +1,12 @@
+import * as Config from "@/modules/config";
+
+import { arbitrumSepolia } from "viem/chains";
+import { NetworkConfig } from "@/modules/chain/NetworkConfig";
+
+export const ChainConfig: NetworkConfig[] = [
+  {
+    ...arbitrumSepolia,
+    biconomyPaymasterApiKey: Config.ArbitrumSepoliaBiconomyPaymasterApiKey,
+    rpcEndpoints: [Config.ArbitrumSepoliaAlchemyRpcEndpoint],
+  },
+];
```

Comment:
This is how we configure chains. We use the native `viem` chains and extend them with our own config where necessary.